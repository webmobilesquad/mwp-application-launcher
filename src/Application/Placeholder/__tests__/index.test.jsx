import render from 'preact-render-spy';
import { Placeholder } from '..';

describe('<Placeholder />', () => {
  test('<Placeholder /> matches snapshot', () => {
    expect(
      render(
        <Placeholder />,
      ),
    ).toMatchSnapshot();
  });
});
