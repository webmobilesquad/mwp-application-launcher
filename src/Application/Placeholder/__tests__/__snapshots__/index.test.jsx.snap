// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Placeholder /> <Placeholder /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<li class="Placeholder">
  <i class="PlaceholderImage"></i>
</li>

`;
